src/components/button/index.tsx

```diff
@@ -1,18 +1,30 @@
 import React, { FC, MouseEvent, ReactNode } from 'react';
 import { Button as ChakraButton } from '@chakra-ui/react';
 
-interface Props {
+type SharedProps = {
   variant?: 'primary' | 'secondary';
   size?: 'md' | 'lg';
   isDisabled?: boolean;
-  onClick: (event: MouseEvent<HTMLElement>) => void;
   children: ReactNode;
-}
+};
+
+type ButtonProps = SharedProps & {
+  type?: 'button';
+  onClick: (event: MouseEvent<HTMLElement>) => void;
```

Comment:
it's not part of this PR but could we use the type from ChakraButtonProps for the onClick? It feels wrong to type it again since it should be the native HTML event type. Maybe `Pick<>` could help here

![Bildschirmfoto 2022-08-08 um 09 04 55](https://user-images.githubusercontent.com/71456764/183359226-187bace2-0f82-43b0-b3d0-2d035182dcb7.png)

Reply:
Looks like it's taken care of by Lorenzo